tripy/tests/frontend/module/test_sequential.py

```diff
@@ -37,6 +37,14 @@ def dict_sequential_network():
     yield tp.Sequential({"layer1": tp.Linear(1, 3), "layer2": tp.Linear(3, 2)})
 
 
+@pytest.fixture
+def mixed_container_sequential_network():
+    yield tp.Sequential(
+        tp.Conv(in_channels=2, out_channels=2, kernel_dims=(1, 1), stride=(1, 1)),
+        lambda x: tp.avgpool(x, kernel_dims=(2, 2), stride=(1, 1)),
```

Comment:
Would be good to have a module after the lambda to make sure indices are skipped correctly.

Reply:
good catch, added now!